content/docs/command-reference/checkenv.md

```diff
@@ -0,0 +1,11 @@
+# checkenv
+
+Check that current environment satisfies object requrements
```

Comment:
Missing period `.`

For things like this, do we have to go to the core repo, make a PR, have it approved, wait for a release, then run this workflow in the docs and check that there are no conflicts vs. doc-specific changes, etc.?

Reply:
Depending on process. For now, you can edit generated sections however you want after the generation. But next time you re-generate it will be lost - you have to change core docs to fix that.
Also, for now scripts just use whatever mlem version you have installed in your env - maybe even not committed. 
I see that fixing small typos like this is a lot faster without full release cycle of the core code, but without it they will not be fixed in the core code.
I guess the compromise looks something like this:
- checkout latest mlem release tag (main branch might have newer features not yet released)
- fix typo and create PR in mlem repo
- generate docs from this code and make a PR in mlem.ai
- probably keep mlem PR open until new release (to make newer fixes possible)
Should suffice for small fixes.

Reply:
where does it take descriptions from (options and the usage section?)? from CLI click speck? or special docstrings? 

(getting back to the question if it generates things primarily from help messages or not)?

Reply:
I checked it one time. I see that it comes from CLI help messages. 

I feels it's good enough to bootstrap and do sanity checks (no missing options). It's fine for usage section probably. It's too limited for the Options section. We should be able to insert examples, admons, lists, there. It should not be just repetition of help messages.

Check DVC / Git option descriptions. One sentence is fine for help, but it's not enough for docs.

Reply:
Agreed. To make this work we can either
1. Have additional hand-crafted section with in-depth option docs (like Description)
2. Have specific syntax inside Options that will tell generator to add lines to generation like
```
[//]: start <option name>
additional stuff
[//]: end <option name>
```

Reply:
We can, but I don't feel it worth the effort, Mike. Again as things settle a bit it's good to have a CI check for example that would notify about discrepancies, but it's really not that problematic to keep docs up to date (at the very least 80%-90% of all the time goes into actually writing stuff that is easy to read and meaningful and it's not something that can be automated).

Reply:
I do like the idea that there's more of a tie between core code and docs so we'll have something to force us to be consistent in language and terminology, for example. In my experience with DVC docs that discrepancy shows up most of the time in the Usage blocks so indeed starting by generating that from a single source (the core repo) could be a good first step.

Not sure about the short description or definition of the commands/methods though (like the one this comment is on): there's a bit more space in the docs format to expand on, or even just add links, tooltips, etc.

Reply:
In this PR I am just trying to recreate what was already in the docs. There is no automation also, so for now you can just manually add something and we can try to either adapt generator to recreate this too or switch it to "check mode" that will not generate stuff (maybe only some sections) but warn if something is missing